src/libs/navigation.c

```diff
@@ -481,6 +486,10 @@ static void _zoom_preset_2(GtkButton *button, gpointer user_data)
 {
   _zoom_preset_change(3);
 }
+static void _zoom_preset_3(GtkButton *button, gpointer user_data)
```

Comment:
I would like to suggest to rename the functions  ``_zoom_preset_1`` to ``_zoom_preset_3`` to something more significant, e.g., ``_zoom_preset_100percent`` and so on.